op-service/eth/ssz_test.go

```diff
@@ -250,21 +250,21 @@ func TestOPB04(t *testing.T) {
 
 func createPayloadWithWithdrawals(w *types.Withdrawals) *ExecutionPayload {
 	return &ExecutionPayload{
-		ParentHash:    common.HexToHash("0x123"),
-		FeeRecipient:  common.HexToAddress("0x456"),
-		StateRoot:     Bytes32(common.HexToHash("0x789")),
-		ReceiptsRoot:  Bytes32(common.HexToHash("0xabc")),
+		ParentHash:    common.HexToHash("0x1234"),
+		FeeRecipient:  common.HexToAddress("0x4567"),
```

Comment:
Hashes and addresses are always a fixed size so there's no need to change these values - they'll always wind up as 32 or 20 bytes long regardless and the existing values parse just fine.